addons/humor/yeetgun/fn_yeetGunFired.sqf

```diff
@@ -0,0 +1,98 @@
+params ["_unit", "_weapon", "_muzzle", "_mode", "_ammo", "_magazine", "_projectile", "_gunner"];
+private _barrelPos = eyePos _unit;
+if (local _unit) then {
+    // Shitty hack for 0 damage bullets, as AI refuse to shoot 0 damage ammo.
+    deleteVehicle _projectile;
+};
+private _range = 85;
+private _pressureBase = 13500; // Wind pressure of 335mph wind
+private _barrelFOV = 45;
+private _nearby = nearestObjects [_unit, ["Static", "CAManBase", "AllVehicles"], _range] - [_unit];
+private _weaponDirVec = _unit weaponDirection currentWeapon _unit;
+{
+    private _target = _x;
+    if (local _target) then {
```

Comment:
Might want to `if (!local _target) then { continue; }` to avoid one level of indent for the rest of the loop.